src/Html.php

```diff
@@ -1262,6 +1262,12 @@ public static function includeHeader(
             $tpl_vars['glpi_request_id'] = \Glpi\Debug\Profile::getCurrent()->getID();
         }
 
+        // Hack as we have no proper way to load a special scss file for a given page...
+        $extra_css_files = $_ENV['extra_css_files'] ?? [];
+        foreach ($extra_css_files as $path) {
+            $tpl_vars['css_files'][] = ['path' => $path];
+        }
+
```

Comment:
I would prefer to not introduce this kind of hack. If plugin developpers starts to use it, it will be a pain to remove later.

You can declare a new "sector" in `$CFG_GLPI['javascript']` and use it here.

Reply:
Done but I still think it is not great.

We are defining a new js library that doesn't exist and associating it to a menu entry that doesn't exist just to load an unrelated scss file... we have to find something better than this at some point because it feel even more hacky this way.

> If plugin developpers starts to use it,

Plugins already have a dedicated hook to inject any css file they want into any page.